index.js

```diff
@@ -47,14 +51,15 @@ module.exports = (target, opts) => {
 			args = args.concat(appArgs);
 		}
 	} else {
+		if (appArgs.length > 0) {
+			args = args.concat(appArgs);
+		}
```

Comment:
Why did you move this?

Reply:
Because unshifting (pushing to the front) of an array that isn't yet created is a bad idea.